packages/amplify-frontend-ios/lib/frontend-config-creator.js

```diff
@@ -35,8 +35,9 @@ function deleteAmplifyConfig(context) {
         const file = path.join(srcDirPath, docsFilePath, `${filename}.${FILE_EXTENSION_MAP[codeGenTarget]}`);
         if (fs.existsSync(file)) fs.removeSync(file);
       });
-
-      fs.removeSync(path.join(srcDirPath, generatedFileName));
+      if (generatedFileName !== '') {
```

Comment:
It's still not a good idea to check for empty string

Reply:
Can you check if it's a directory or make it sure it's a file?

Reply:
Checking if empty string is the only way, because the generatedfilename by itself is **not** a path

Reply:
It is not bad practice to make a check for empty string